README.md

```diff
@@ -142,12 +142,14 @@ For that, you will need to modify the `docker-compose.yml` file to mount the `sr
 
 For example:
 ```
-service:
+services:
   bookbrainz-site:
     command: npm run debug
     volumes:
+      - "./config/config.json:/home/bookbrainz/bookbrainz-site/config/config.json:ro"
```

Comment:
Thought this will help prevent replacing *config* line with *src* line

Reply:
Good thinking !